unlock-app/src/hooks/useProvider.ts

```diff
@@ -44,13 +44,7 @@ export const useProvider = (config: any) => {
 
   const createWalletService = async (provider: any) => {
     const _walletService = new WalletService(config.networks)
-    let _network = 1 // default
-    try {
-      _network = await _walletService.connect(provider)
-    } catch (error) {
-      console.log(error)
-    }
```

Comment:
Was whatever issue caused this to throw resolved? 🤔 I'm not sure I see any change above.

Reply:
I actually could not find what it was for so I removed it... we catch in other places anyway.